src/meta-srv/src/service/admin/maintenance.rs

```diff
@@ -60,23 +73,24 @@ impl MaintenanceHandler {
                 err_msg: "'enable' must be 'true' or 'false'",
             })?;
 
-        let response = if enable {
+        if enable {
             self.manager
                 .set_maintenance_mode()
                 .await
                 .context(MaintenanceModeManagerSnafu)?;
-            "Maintenance mode enabled"
+            info!("Set maintenance mode to true");
```

Comment:
The logs will be better:

> Enable the maintenance mode.

Reply:
resolved in [7f09c0c](https://github.com/GreptimeTeam/greptimedb/pull/6022/commits/7f09c0c589f45937ca55ba3d05e3a40ffdc118b3)